frontend/views/containers/chatroom/MessageActions.vue

```diff
@@ -276,23 +276,25 @@ export default ({
   }
 }
 
-.c-menu {
+.c-message-menu {
   position: absolute;
   right: 0;
   top: 0;
   height: 100%;
   width: auto;
+}
 
+.c-menu {
   .c-content {
     @include tablet {
       width: 100%;
       left: auto;
       right: 0.5rem;
```

Comment:
DOM structure here seems to be re-organized here. Not a serious thing but I see a minor alignment bug introduced here:

<img width="380" alt="image" src="https://github.com/user-attachments/assets/56a76a79-50ac-49ed-8743-677d88a97281" />

changing `right: 0;` here fixes it.